pages/linux/dnsrecon.md

```diff
@@ -0,0 +1,32 @@
+# dnsrecon
+
+> DNS enumeration tool.
+> More information: <https://github.com/darkoperator/dnsrecon>.
+
+- Scan a domain and save the results to a SQLite database:
+
+`dnsrecon --domain {{example.com}} --db {{path/to/database.sqlite}}`
+
+- Scan a domain, specifying the nameserver and performing a zone transfer:
+
+`dnsrecon --domain {{example.com}} --name_server {{nameserver.example.com}} --type axfr`
+
+- Scan a domain, using a dictionary of subdomains and hostnames for bruteforcing:
+
+`dnsrecon --domain {{example.com}} --dictionary {{path/to/dictionary.txt}} --type brt`
+
+- Scan a domain, performing a reverse lookup of IP ranges from the SPF record and saving the results to a JSON file:
```

Comment:
There is no need to mention "Scan a domain" after the first 3 times. It becomes implicit.